homeassistant/components/media_player/plex.py

```diff
@@ -4,35 +4,68 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/media_player.plex/
 """
+from datetime import datetime, timedelta
 import json
 import logging
 import os
-from datetime import timedelta
 from urllib.parse import urlparse
 
-import homeassistant.util as util
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+import asyncio
```

Comment:
Can you include a screenshot of how you are seeing the errors?  When I click on the build jobs, I see a bombardment of lines, what look like sql queries, errors that don't pertain to the code, etc?  Thanks.